opta/commands/config.py

```diff
@@ -0,0 +1,62 @@
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.azure import Azure
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.exceptions import UserErrors
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    """
+    (Beta) Manage opta configurations.
```

Comment:
the terraform generator is in beta, because we have not yet tested all the modules, but this feature does not need to be tested for each module, do we need to be _beta_?

Reply:
Removed it.